.github/workflows/publish-release.yml

```diff
@@ -20,7 +20,10 @@ jobs:
               env:
                 packageUser: ${{ github.actor }}
                 packagePAT: ${{ secrets.GITHUB_TOKEN }}
-              run: ./gradlew build -x check -x test
+              run: |
+                git config --global user.name ${{ secrets.BALLERINA_BOT_USERNAME }}
+                git config --global user.email ${{ secrets.BALLERINA_BOT_EMAIL }}
+                ./gradlew build -x check -x test
```

Comment:
Is this module has a `check` task in the gradle build? If not, `-x check` should be removed.

Reply:
This module has a `check` task.